packages/ui/src/components/InputLabel/styles.module.css

```diff
@@ -1,6 +1,5 @@
 .input-label {
     display: flex;
-    padding: 0 var(--go-ui-spacing-2xs);
```

Comment:
Shouldn't we be adding instead of removing the padding? Has this been defined somewhere else now?

Reply:
It was removed to maintain label and value spacing as mentioned by the auditor